src/goto-instrument/accelerate/accelerate.cpp

```diff
@@ -651,14 +673,16 @@ int acceleratet::accelerate_loops()
 
 void accelerate_functions(
   goto_modelt &goto_model,
-  bool use_z3)
+  bool use_z3,
```

Comment:
If you want a refactor; the use Z3 option should not be passed around like this.